sql/step_a107_orthanc_config.sql

```diff
@@ -0,0 +1,35 @@
+-- Orthanc Config Access And lien Orthanc Patient to patient int OH
+
+-- Create new table Orthanc config user access
+CREATE TABLE OH_ORTHANC_CONFIG (
```

Comment:
Lower name the table's name. 
If this table will be used to store users' credentials to access Orthanc, the name `oh_orthanc_user` is more appropriated